overlay_new.py

```diff
@@ -62,23 +62,20 @@ def data_field_coord(x, y):
                 data_field_coord(3, 1),
             ),
             CentrePower(self.width, self.height),
-            Message(),
+            DAShboardMessage(),
+            DASDisconnectMessage(self.client),
         ]
-        self.das_disconnect_message = DASDisconnectMessage()
 
     def on_connect(self, client, userdata, flags, rc):
         print("Connected with rc: {}".format(rc))
```

Comment:
I've realised that we print this message for all overlays, and then do nothing else important on connection. Can you move this into the default implementation of `on_connect`, and then make the function non-abstract? Seems like we don't really have a need for it anymore with all this code cleaning up :)